core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -825,6 +826,11 @@ class SparkContext(config: SparkConf) extends Logging {
   def getLocalProperty(key: String): String =
     Option(localProperties.get).map(_.getProperty(key)).orNull
 
+  /** Set the UUID of the Spark session that starts the current job. */
+  def setSparkSessionUUID(uuid: String): Unit = {
+    setLocalProperty(SparkContext.SPARK_SESSION_UUID, uuid)
```

Comment:
SparkContext is only one. How does this relate to Spark Session UUID?

Reply:
Here I'm trying to inject session UUID into new jobs, so from Session we could cancel them all. Do you have concerns about this approach or the naming?

Reply:
Spark core has to be separated from Spark SQL, and now we're putting SQL stuff into Spark core. Can we avoid this?

Reply:
How about removing this API and using `setLocalProperty` instead?
Also the `getSparkSessionUUID` API in ActiveJob will be removed.

Reply:
I stopped mentioning "SparkSession" in Core and use setLocalProperty instead.